.github/workflows/containers.yaml

```diff
@@ -3,8 +3,6 @@ name: Containers - Build and push images
 
 "on":
   push:
-    branches:
-      - main
```

Comment:
If we're going to push on non-main, then you should probably change the logic to update latest below - since that'll automatically get updated for other branches.

Reply:
Sorry, Im not sure I'm following - able to expalin more?

Reply:
The "latest" tag is automatically updated in some situations. So if we start pushing for non-main branches then we should probably just use "main" for the main branch.

Reply:
To me, latest is the lastest on main.

Reply:
The thing is that the registry will update the latest tag automatically when new builds are pushed - and then it won't match main anymore.

Reply:
I can't say ive seen that behaviour. Is there any examples/docs of that happening?

Reply:
See e.g. https://www.docker.com/blog/docker-best-practices-using-tags-and-labels-to-manage-docker-image-sprawl/

Reply:
Gotcha - godo to know!

Updated